{{ cookiecutter.package_name }}/setup.py

```diff
@@ -1,17 +1,15 @@
 #!/usr/bin/env python
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+# Note: This file needs to be Python 2 / <3.6 compatible, so that the nice
+# "This package only supports Python 3.x+" error prints without syntax errors etc.
```

Comment:
Maybe put it around the configparser import since that's where it matters?

Reply:
Well that's the bit that matters at the moment, but the whole file must be parseable on py2. i.e. no f-strings or other fun things.

Reply:
Ok cool - can we add some kind of CI test to make sure it stays that way?

Reply:
maaayybbeee?! :stuck_out_tongue: